fcrepo-http-commons/pom.xml

```diff
@@ -112,14 +119,18 @@
     </dependency>
     <dependency>
       <groupId>org.glassfish.jersey.ext</groupId>
-      <artifactId>jersey-spring3</artifactId>
+      <artifactId>jersey-spring5</artifactId>
       <scope>test</scope>
       <version>${jersey.version}</version>
     </dependency>
     <dependency>
       <groupId>org.glassfish.jersey.core</groupId>
       <artifactId>jersey-common</artifactId>
-      <version>${jersey.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.hk2</groupId>
+      <artifactId>hk2-api</artifactId>
+      <version>2.6.1</version>
```

Comment:
Oops I missed one. We can handle this later or I can submit another PR to this one to define this dependency at the root.

Reply:
Another PR to this PR would be good. They should all get squashed before going into `master` anyways.